app/components/Layout.tsx

```diff
@@ -40,18 +40,17 @@ export function Layout({ back, children, title }: LayoutProps) {
 						About
 					</InternalAnchor>
 					<span className={styles.dot} />
-					<InternalAnchor reloadDocument to="/code-of-conduct" variant="heavy">
-						Code of Conduct
-					</InternalAnchor>
-				</div>
```

Comment:
[Request] I think we should leave the code of conduct on the main nav. I know a lot of people who definitively feel less comfortable going to events without one linked on at least the homepage, if not every page. And I've anecdotally heard of folks using _"it was hard to find!"_ as an excuse for misbehaving. Granted, we always announce it in opening announcements and can put it on a table display - but there's time before that when people can "miss" it.

Reply:
Mmh, ok. I don't like the nav split onto 2 lines but oh well. Re-added. Let me know what you think.

Reply:
I also don't like the nav split onto 2 lines 😂... maybe there's a compromise we could get to? Shrinking the text to something like _CoC_ or _Rules_?

Reply:
I don't think either are desirable: "CoC" is not a very widespread acronym among the gen pop and "Rules" is not very representative, especially if the initial argument was that the code of conduct is hard to find.
Besides, I don't think the spacing makes sense:
<img width="785" alt="Screenshot 2025-01-30 at 20 21 44" src="https://github.com/user-attachments/assets/c5169b44-4390-4fc2-b535-b518d901f1a7" />

I fixed the CSS to support 2 lines a little better so it's fine as is IMO. If 1 line is the priority, then I think the link to the code of conduct needs to go.

Reply:
Makes sense. In that case I defer to you :) whatever you prefer!

Reply:
I'm fine either way